ipn/ipnlocal/network-lock.go

```diff
@@ -522,8 +534,15 @@ func (b *LocalBackend) NetworkLockModify(addKeys, removeKeys []tka.Key) (err err
 	if b.tka == nil {
 		return errNetworkLockNotActive
 	}
+	var nlPriv key.NLPrivate
+	if p := b.pm.CurrentPrefs().Persist(); p != nil {
+		nlPriv = p.NetworkLockKey
+	}
+	if nlPriv.IsZero() {
+		return errMissingNetmap
```

Comment:
Either return `errNetworkLockNotActive` or customize the error, because the invariant of `b.tka != nil` with `nlPriv.IsZero()` should not be possible (or at least I really hope so).

Reply:
Good point, I reordered the checks so that we verify `nlPriv.IsZero` before we check for `b.tka == nil`